galaxyproject/galaxy
# Visualizations, trackster: allow the user to add the dataset launched…

… from as a new track in an existing trackster visualization; add utils/query-string-parsing.js for pulling query params from the query string

choose_existing_or_new is a modified version of the original create_trackster_action_fn which lived in hda-edit back in 2013. It's now adapted to work with view_new and view_existing.

Steps to test (assuming you've set up trackster):
- click on the visualization button of a dataset that trackster can use, then click 'Trackster'
- the trackster page should load and a dialog should give you the choice between a new visualization or an 'saved' one
- clicking 'saved' will open a new dialog with a list of saved trackster sessions that share the dbkey of your dataset
- clicking 'new' will open the dialog that creates a new track browser

@jgoecks @jennaj 
https://trello.com/c/lx0jYO2k
